raphtory-graphql/src/model/mod.rs

```diff
@@ -239,6 +244,22 @@ impl Mut {
         new_subgraph.update_constant_properties([("path", Prop::Str(path.clone().into()))])?;
         new_subgraph.update_constant_properties([("isArchive", Prop::U8(is_archive))])?;
 
+        // Temporary hack to allow adding of arbitrary maps of data to nodes
+        for node in new_subgraph.nodes() {
+            if !node_map.contains_key(&node.name().to_string()) {
+                println!("Could not find key! {} {:?}", node.name(), node_map);
+                panic!()
+            }
+            let node_props = node_map
+                .get(node.name().to_string().as_str())
+                .ok_or(format!(
+                    "Could not find node {} in provided map",
+                    node.name()
+                ))?;
+            let node_props_as_str = node_props.as_str().ok_or("node_props must be string")?;
+            node.update_constant_properties([("props", node_props_as_str)])?;
+        }
```

Comment:
you could have made this less hacky by making the properties an object? So you can have properties other than `"props"` and should be able to have other types as well I assume?

Reply:
i.e., why not accept strings like this? "{"node1": {"x": 1, "y": 1}, "node2: {"x": 2, "y": 2}}"?